build.sbt

```diff
@@ -44,6 +52,7 @@ lazy val root = Project(
     // Unidoc doesn't like macros
     unidocProjectExcludes := Seq(parsing)
   )
+  .settings(dontPublishSettings)
```

Comment:
This should not be needed because of the `.enablePlugins(..., NoPublish)` above.